doc/installation/system/inifile.rst

```diff
@@ -25,6 +25,8 @@ The file should contain the following contents::
    SUPERUSER_REALM = ['super', 'administrators']
    # Your database
    SQLALCHEMY_DATABASE_URI = 'sqlite:////etc/privacyidea/data.sqlite'
+   # Set maximum identifier length to 128
+   SQLALCHEMY_ENGINE_OPTIONS = {"max_identifier_length": 128}
```

Comment:
Please add this as comment

    # SQLALCHEMY_ENGINE_OPTIONS = {"max_identifier_length": 128}

Reply:
Fixed in 311611ae32b43837bfd4427d1cd6766ab188361c.